backend/src/main/java/com/cgi/retrospecto/backend/poker/service/RoomService.java

```diff
@@ -0,0 +1,23 @@
+package com.cgi.retrospecto.backend.poker.service;
+
+import com.cgi.retrospecto.backend.poker.domain.Room;
+import com.cgi.retrospecto.backend.poker.domain.Story;
+import com.cgi.retrospecto.backend.poker.domain.Vote;
+import com.cgi.retrospecto.backend.poker.exception.RoomNotFoundException;
+import com.cgi.retrospecto.backend.poker.exception.StoryNotFoundException;
+
+public interface RoomService {
```

Comment:
Same comment as in the controller and interface since there i only one service and these methods will not be used anywhere else an interface is overcomplicating the code in my opinion.

Reply:
Removed the interface.